BaseUI/src/components/FilledButton/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {TouchableOpacity, Text} from 'react-native';
+import FilledButtonStyles from './styles';
+
+interface FilledButtonProps {
+  title: string;
+  onPress: () => void;
+  disabled?: boolean;
+  textColor?: string;
+  backgroundColor?: string;
+}
+
+class FilledButton extends React.Component<FilledButtonProps, {}> {
+  render(): React.ReactNode {
+    return (
+      <TouchableOpacity
+        style={[
+          FilledButtonStyles.touchableOpacityStyle,
+          this.props.backgroundColor
```

Comment:
Here you can add a background color like this:

{ backgroundColor: this.props.backgroundColor }